javascript/ql/src/semmle/javascript/frameworks/Express.qll

```diff
@@ -479,6 +479,17 @@ module Express {
           methodName = "header"
         )
         or
+        exists (DataFlow::PropRead headers |
+          // `req.headers.name`
+          kind = "header" and
+          headers.accesses(request, "headers") and
+          this = headers.getAPropertyRead(_))
```

Comment:
Perhaps use `getAPropertyRead()` instead (without the don't-care argument) so it also considers dynamic property reads?

Reply:
👍 Thanks for spotting that.